### DISCUSSION TITLE
请问有Reality和Naiveproxy共存的方案？

### DISCUSSION THREAD

**Discussion by RaigoTsui** [lxhao61/integrated-examples#190] [Q&A] [answered]
如题，有没有以Reality监听443，然后回落到Naiveproxy的方案可用，还是说Reality+caddy的方案就可以，不大懂

**Comment by lxhao61** [comment, marked as answer]
参考 [Xray(M+H+K+A)+Caddy(N+T)](https://github.com/lxhao61/integrated-examples/tree/main/Xray(M%2BH%2BK%2BA)%2BCaddy(N%2BT)) 示例，删减即可。

**Reply by RaigoTsui** [reply]
请问这个配置能进行建站吗？还是必须上nginx
